packages/wxt/src/core/utils/strings.ts

```diff
@@ -5,6 +5,14 @@ export function kebabCaseAlphanumeric(str: string): string {
     .replace(/\s+/g, '-'); // Replace spaces with hyphens
 }
 
+/**
+ * Return a safe variable name for a given string.
+ */
+export function safeVarName(str: string): string {
+  // _ prefix to ensure it doesn't start with a number
+  return '_' + kebabCaseAlphanumeric(str.trim()).replace('-', '_');
+}
```

Comment:
I have two main use cases:

1. Scripts injected into the main page (to gain access to global objects
2. Scripts run in Web Worker

For 1, I can see there might be a chance of naming collisions and for 2 it would just produce syntax errors with evaluating undeclared variables generated here by WXT.

Let me see if I can put up a minimal reproduction for this.